src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -130,6 +135,8 @@ export const ReviewRoute: FC<ReviewProps> = props => {
           o => o.lineItems?.edges?.[0]?.node?.artwork?.slug
         )
 
+        requestIdentityVerificationIfNecessary(order)
```

Comment:
this might need an `await requestIdentityVerificationIfNecessary(order)`?

Reply:
That's right. I removed that by mistake. Thank you.